manifest.webapp

```diff
@@ -43,6 +43,17 @@
       "type": [
         "io.cozy.apps"
       ]
+    },
+    {
+      "action": "OPEN",
```

Comment:
To me `open io.cozy.apps` should open the cozy-app, not redirecting on the store to install it, so I don't think this is the right verb.

Why do we create a new intent? Why don't we use the previous `INSTALL` one in which we add the `category` data?

Reply:
Je suis d'accord que l'action `open` n'est pas la mieux choisie, mais l'action `install` ne répond pas aux besoins.
Que ce soit avec une `category` ou `slug`, en effet avec ce dernier nous ne souhaitons pas arriver sur la page des permissions mais sur la page des détails.

On peut imaginer utiliser l'intent actuel, mais avec deux autres options (`category` et ??), pour afficher une autre page que l'actuelle même avec un `slug` 🤷

Reply:
targetedPage: details | permissions ?

Reply:
Mince, ta réponse ne c'est pas affichée sur le moment...
J'ai opté pour une autre approche

Reply:
Approche revue